pynhm/hydrology/PRMSCanopy.py

```diff
@@ -135,6 +133,26 @@ def get_variables() -> tuple:
             # i defined metadata for it in a very adhoc way
         )
 
+    @staticmethod
+    def get_init_values() -> dict:
+        """Get canopy inital values
+
+        Returns:
+            dict: inital values for named variables
+        """
+
+        return {
```

Comment:
almost all of these were implicitly being inited to zero, which seems obscure to me

Reply:
Agreed that it is obscure.  But I think intcp_stor is the only one that matters, and it starts at zero in prms as far as I can tell.

Reply:
Trying to avoid "it starts at zero .. as far as I can tell". I think if there is an implicit default value it should be nan. But I'm against that